include/ada/url-inl.h

```diff
@@ -188,7 +188,7 @@ inline void url::copy_scheme(const ada::url &u) {
     if (has_credentials()) {
       output += username;
       if (!password.empty()) {
-        output += ":" + get_password();
+        output += ":" + std::string(get_password());
```

Comment:
hmmm?

Reply:
It was returning string& before now string view. I'll land these changes in a different PR to improve reviews